components/plat/pc99/plat/components/serial_com1/Serial.camkes

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2018, Data61, CSIRO (ABN 41 687 119 230)
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+component Serial {
```

Comment:
Can't we avoid code duplication and make this be generic? Have a macro that take the different configs as parameters.

Reply:
This is ugly, but it's probably as good as we can get for now.